app/assets/javascripts/blacklight/facet_load.js

```diff
@@ -0,0 +1,15 @@
+(function($) {
+  Blacklight.onLoad(function() {
```

Comment:
@jkeck i know we're run into this elsewhere -- any idea how to deal with it? Lots of js requires?